Asynch

Lukease opened the pull request.

Commit:
initial commit

Commit:
Finish Project

Commit:
remove comment,
change name from number to scoreNumber,
change let to const,
create new function createCard(),
delete () from click function,
use concat on clickedList.

Commit:
used destructuring,
delete cardsArray,
create new function addHiddenImages(),
create new array sortedArrayOfImages,
use forEach on sortedArrayOfImages.

Commit:
add spaces.

Commit:
jquery

Commit:
jquery

Commit:
jquery

Commit:
remove enter,
change <div></div> to <div>.

Commit:
create function async.

jpudysz commented on cats/index.html:
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
remove all images from the project

jpudysz commented on cats/main.js:
```diff
@@ -0,0 +1,119 @@
+            console.log(error)
+        })
+
+    const addHiddenImages = () => {
+        sortedArrayOfImages = imagesArray.concat(imagesArray)
+        sortedArrayOfImages.sort((a, b) => 0.5 - Math.random())
+
+        return sortedArrayOfImages
+    }
```
this function doesnt make sense if you're invoking it below, simply remove function

jpudysz reviewed (changes requested):
What about loader and error message for the user?

Commit:
add loader,
add alert to catch,
remove all images from project,
remove function addHiddenImages.

jpudysz commented on cats/main.js:
```diff
@@ -91,20 +94,15 @@ const downloadCats = async () => {
         })
         .catch(error => {
-            console.log(error)
+            alert(error)
```
show it in HTML, not with alert

jpudysz commented on cats/main.js:
```diff
@@ -91,20 +94,15 @@ const downloadCats = async () => {
         .then(res => {
             res.forEach(object => {
                 imagesArray = imagesArray.concat(object.url)
+                loader.remove()
```
or use finally

Commit:
add finally,
create new div in catch.

Commit:
add finally,
create new div in catch.
